.github/workflows/electron-build.yml

```diff
@@ -47,17 +47,16 @@ jobs:
         run: |
           aws s3 sync s3://lisk-desktop-files/fonts ./setup/react/assets/fonts/
 
+      - name: Update package.json
+        if: startsWith(matrix.os, 'macos')
+        run: |
+          jq '.build.mac.notarize.teamId = "${{ secrets.APPLE_TEAM_ID }}"' package.json > package_temp.json
+          mv package_temp.json package.json
```

Comment:
Why this temporary file move?

Reply:
It's basically overwriting the package.json from temp